### DISCUSSION TITLE
Multiple inputs to @du.callback ?

### DISCUSSION THREAD

**Discussion by VirajOke** [np-8/dash-uploader#109] [Q&A] [answered]
Can I pass multiple inputs to @du.callback similar to @app.callback? I was looking to use drop-down properties in my du.callback block for a function call. Thank you in advance! 

```
@du.callback(
    output= [Output('func-call', 'children'),Output("analyze-val", "disabled")],
    id= "dash-uploader", <can I add inputs here?>
)
```

**Comment by fohrloop** [comment, marked as answer]
Hi @VirajOke 

Thanks for the question! Does discussion on [issue 104](https://github.com/np-8/dash-uploader/issues/104) help you? I'll have to take a better look on this when I have time.

**Reply by VirajOke** [reply]
Thank you so much for your response. I tried the suggested option 'C' from the discussion, and it worked perfectly!
